packages/api/src/auth/auth.controller.spec.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-underscore-dangle */
+import { Test } from '@nestjs/testing';
+import httpMocks from 'node-mocks-http';
+
+import { env } from '@/common/env';
+
+import { AuthController } from './auth.controller';
+
+describe('Auth controller', () => {
+  let controller: AuthController;
+
+  beforeEach(async () => {
+    const module = await Test.createTestingModule({
+      controllers: [AuthController],
+    }).compile();
+
+    controller = await module.resolve(AuthController);
+  });
+
+  describe('logout', () => {
+    it('Returns true for successful requests', async () => {
```

Comment:
I don't know why we need response body here. true/ false do not have some exact meaning. 
I don't see any test for not successful request.
For me response codes are enough for indicating that request was successful (200), or not (400 for example).

Reply:
You're right. I will remove the return and send 204 to specify that it's a successful response but with no content.

When it comes to testing non-successful requests I don't think there is a need to do so. This controller only passes errors created by services it's using - that's the default nest behavior.  In my opinion, we should test errors only in places where we throw them.